wordle-web/pages/game.vue

```diff
@@ -52,7 +72,15 @@
       <v-row justify="center" class="mt-10">
         <v-alert v-if="wordleGame.gameOver" width="80%" :type="gameResult.type">
           {{ gameResult.text }}
-          <v-btn class="ml-2" @click="resetGame"> Play Again? </v-btn>
+          <v-btn class="ml-2" color="primary" @click="resetGame">
+            Play Again?
+          </v-btn>
+          <v-btn color="primary" :disabled="saved" @click="checkForName()">
+            Save Score?
+          </v-btn>
+          <v-btn color="primary" :disabled="unsure" @click="areYouSure()">
+            Are you sure?
+          </v-btn>
```

Comment:
This appears to display behind the win alert, making it very difficult to see

Reply:
The 'Are you sure?' is meant as a confirmation for if the person has the name 'Guest'. If both buttons are disabled, than the game has been saved. If you choose a name before saving, the second button does not appear